client/packages/system/src/Report/api/hooks/utils/usePrintReport.ts

```diff
@@ -12,6 +12,33 @@ type PrintReportParams = {
   dataId: string;
 };
 
+const setClose = (frame: HTMLIFrameElement) => () => {
+  document.body.removeChild(frame);
+};
+
+const setPrint = (frame: HTMLIFrameElement) => () => {
+  const { contentWindow } = frame;
+  if (contentWindow) {
+    contentWindow.onbeforeunload = setClose(frame);
+    contentWindow.onafterprint = setClose(frame);
+    contentWindow.focus(); // Required for IE
```

Comment:
comment is ok here if we would do the same for desktop, right?

Reply:
ah, yes. the comment was because I copied the basis of the code. I thought it would be good to leave just in case we do use it for the web version